amplifier panic on amp stack rm

I ran into a crash of `amplifier` while trying to delete a stack:
```
$ amp -k stack rm pi
[user su @ localhost:50101]
Error: transport is closing
```

`amplifier` logs:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x10b8761]

goroutine 1952 [running]:
github.com/appcelerator/amp/api/rpc/stack.convertError(0x0, 0x0, 0xc4209a24b0, 0xc42065d100)
	/home/bq/go/src/github.com/appcelerator/amp/api/rpc/stack/stack.go:35 +0xc1
github.com/appcelerator/amp/api/rpc/stack.(*Server).Remove(0xc4203d2e10, 0x7ff297432300, 0xc4209a24b0, 0xc4202c0060, 0xc4203d2e10, 0x0, 0x0)
	/home/bq/go/src/github.com/appcelerator/amp/api/rpc/stack/stack.go:139 +0x423
github.com/appcelerator/amp/api/rpc/stack._Stack_Remove_Handler.func1(0x7ff297432300, 0xc4209a24b0, 0x1323660, 0xc4202c0060, 0xc4209a24b0, 0x0, 0x0, 0xc420372360)
	/home/bq/go/src/github.com/appcelerator/amp/api/rpc/stack/stack.pb.go:439 +0x86
github.com/appcelerator/amp/api/auth.(*Interceptors).Interceptor(0xc42000c0f0, 0x7ff297432300, 0xc4209621b0, 0x1323660, 0xc4202c0060, 0xc420350ce0, 0xc420350d00, 0x20, 0x12f0000, 0x1, ...)
	/home/bq/go/src/github.com/appcelerator/amp/api/auth/auth.go:73 +0x79
github.com/appcelerator/amp/api/auth.(*Interceptors).Interceptor-fm(0x7ff297432300, 0xc4209621b0, 0x1323660, 0xc4202c0060, 0xc420350ce0, 0xc420350d00, 0x0, 0xc420b499b8, 0x40f4c8, 0x50)
	/home/bq/go/src/github.com/appcelerator/amp/cmd/amplifier/server/server.go:113 +0x73
github.com/appcelerator/amp/api/rpc/stack._Stack_Remove_Handler(0x1333ca0, 0xc4203d2e10, 0x7ff297432300, 0xc4209621b0, 0xc4202395e0, 0xc420375af0, 0x0, 0x0, 0x0, 0x0)
	/home/bq/go/src/github.com/appcelerator/amp/api/rpc/stack/stack.pb.go:441 +0x177
github.com/appcelerator/amp/vendor/google.golang.org/grpc.(*Server).processUnaryRPC(0xc420372360, 0x1cd0380, 0xc420bb2000, 0xc420396c60, 0xc4203d2ea0, 0x1d40bd0, 0xc420962210, 0x0, 0x0)
	/home/bq/go/src/github.com/appcelerator/amp/vendor/google.golang.org/grpc/server.go:753 +0xcd0
github.com/appcelerator/amp/vendor/google.golang.org/grpc.(*Server).handleStream(0xc420372360, 0x1cd0380, 0xc420bb2000, 0xc420396c60, 0xc420962210)
	/home/bq/go/src/github.com/appcelerator/amp/vendor/google.golang.org/grpc/server.go:955 +0x15a0
github.com/appcelerator/amp/vendor/google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc4201804e0, 0xc420372360, 0x1cd0380, 0xc420bb2000, 0xc420396c60)
	/home/bq/go/src/github.com/appcelerator/amp/vendor/google.golang.org/grpc/server.go:517 +0xa9
created by github.com/appcelerator/amp/vendor/google.golang.org/grpc.(*Server).serveStreams.func1
	/home/bq/go/src/github.com/appcelerator/amp/vendor/google.golang.org/grpc/server.go:518 +0xa1

```

--- a/api/rpc/stack/stack.go
+++ b/api/rpc/stack/stack.go
@@ -136,7 +136,7 @@ func (s *Server) Remove(ctx context.Context, in *RemoveRequest) (*RemoveReply, e
 		return nil, convertError(dockerErr)
 	}
 	if storageErr != nil {
-		return nil, convertError(dockerErr)
+		return nil, convertError(storageErr)
 	}
 
 	log.Infof("Stack %s removed", in.Stack)

